spec/requests/api/querying_spec.rb

```diff
@@ -323,9 +323,21 @@ def create_vms_by_name(names)
 
       run_get vms_url, :expand => "resources", :attributes => "name,vendor"
 
-      expect_query_result(:vms, 1, 1)
```

Comment:
I don't want to lose these.  query result structure (name/count/subcount/...) being checked in a common method. Maybe there's a different way to represent below in expected = { } without having to respecify them in each test.